src/models/Requisite.php

```diff
@@ -48,18 +49,26 @@ public function rules()
             [['invoice_name', 'acceptance_name', 'contract_name', 'probation_name', 'internal_invoice_name', 'proforma_name'], 'safe'],
             [['balance', 'balances'], 'safe'],
         ]);
+        $templatesTypes = [];
+        foreach (self::getTemplatesTypes() as $templatesType) {
+            $templatesTypes[] = "{$templatesType}_id";
+        }
+        $rules[] = [$templatesTypes, 'string', 'on' => 'create'];
+        $rules[] = [['invoice_id'], 'required', 'on' => 'create'];
+
+        return $rules;
     }
 
     public function isRequisite()
     {
-        return (boolean) $this->is_requisite;
+        return (boolean)$this->is_requisite;
```

Comment:
```suggestion
        return (bool)$this->is_requisite;
```
чтоб не было вразнобой